src/main/java/com/internet/shop/controller/shoppingcart/ShoppingCartController.java

```diff
@@ -0,0 +1,26 @@
+package com.internet.shop.controller.shoppingcart;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.service.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/shoppingCart")
+public class ShoppingCartController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
```

Comment:
```suggestion
    private static final Injector INJECTOR = Injector.getInstance("com.internet.shop");
```
аналогічно в інших класах, де є така помилка